tests/e2e/tests/dashboard/dashboard_test.go

```diff
@@ -542,6 +546,71 @@ func logMixerMetrics(t *testing.T, service string, port int) {
 	t.Logf("GET http://%s.%s:%d/metrics:\n%v", service, ns, port, resp)
 }
 
+func dumpProxyStats(t *testing.T) {
```

Comment:
Please move all this to some common place, and include it in the new dump script we provide to the users...
No more test-specific infra.

Reply:
Could you give me a pointer where is the new dump script?